services/apps/premium/members_enrichment_worker/src/activities/getMembers.ts

```diff
@@ -0,0 +1,49 @@
+import { IMember } from '@crowd/types'
+
+import { svc } from '../main'
+
+/*
+getMembers is a Temporal activity that retrieves all members available for
+enrichment. Member must have one of GitHub username or email address, must not
+have been enriched in the past 90 days, and must be part of tenant with a plan
+allowing this feature. We limit to 50 members per workflow to not overload
+external APIs.
+*/
+export async function getMembers(): Promise<IMember[]> {
+  let rows: IMember[] = []
+
+  try {
+    rows = await svc.postgres.reader.connection().query(
+      `SELECT
+          members."id",
+          members."displayName",
+          members."attributes",
+          members."emails",
+          members."contributions",
+          members."score",
+          members."reach",
+          members."tenantId",
+          jsonb_object_agg(mi.platform, mi.username) as username
+        FROM members
+        INNER JOIN tenants ON tenants.id = members."tenantId"
+        INNER JOIN "memberIdentities" mi ON mi."memberId" = members.id
+        WHERE tenants.plan IN ('Scale', 'Enterprise')
+        AND (
+          members."lastEnriched" < NOW() - INTERVAL '90 days'
+          OR members."lastEnriched" IS NULL
+        )
+        AND (
+          mi.platform = 'github'
+          OR array_length(members.emails, 1) > 0
+        )
+        AND tenants."deletedAt" IS NULL
+        AND members."deletedAt" IS NULL
+        GROUP BY members.id
+        LIMIT 50;`,
```

Comment:
If we are doing 50 at a time we should prioritize with order. I think activity count would be the best priority. Otherwise, the most important members might take long to be enriched.